README.md

```diff
@@ -0,0 +1,56 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magician
+
+> math_magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to: Make simple calculations and read a random math-related quote.
+
+## Tools & Technologies
+
+- React.
+- Eslint.
+- Stylelint.
+
+## Live Demo 
+
+Coming Soon
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+```
+git clone git@github.com:OmarMWarraich/math-magicians.git
+cd math-magician
+npm install
+npm start
+```
+
+## Pre-requisites
+
+- node.js
+- npm
+
+## Author
+
+👤 **Author**
+
+- GitHub: [@OmarMWarraich](https://github.com/OmarMWarraich)
+- Twitter: [@omarwarraich1](https://twitter.com/@omarwarraich1)
+- LinkedIn: [Omar Mohammad](https://www.linkedin.com/in/omar-mohammad-a9902847/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thank You [Microverse](www.microverse.org) team.
+- Thanks to everyone who will get time to check this code and suggest any [issues](https://github.com/OmarMWarraich/math-magicians/issues)
+
```

Comment:
- [ ] As requested by the previous reviewer, Please add an MIT license to your project and link that in the README file in the `License` section. You can use this [template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)